bin/ssh_scan

```diff
@@ -35,8 +35,15 @@ opt_parser = OptionParser.new do |opts|
       exit
     end
     File.open(file).each do |line|
-      line.chomp.split(',').each do |ip|
+      line.chomp.split(',').each do |ips|
```

Comment:
Naming variables are getting really confusing. So after splitting about comma we would get a socket, so instead  of ips you could name it socket.